discovery-provider/src/schema_track_simple.json

```diff
@@ -0,0 +1,278 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$ref": "#/definitions/Track",
+    "definitions": {
+        "Track": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "blockhash": {
+                    "type": "string"
+                },
+                "blocknumber": {
+                    "type": "integer"
+                },
+                "track_id": {
+                    "type": "integer"
+                },
+                "is_current": {
+                    "type": "boolean"
+                },
+                "is_delete": {
+                    "type": "boolean"
+                },
+                "owner_id": {
+                    "type": "integer"
+                },
+                "route_id": {
+                    "type": "string"
+                },
+                "title": {
+                    "type": ["string", "null"],
```

Comment:
what is the array for?

Reply:
@raymondjacobson is correct. this means that the title field can be nullable or a string. i was going off of models.py where the title field did not have `nullable=False` set. we can definitely revert this to only accept type of string

Reply:
also @raymondjacobson i will look into adding using defaults in the jsonschema for the next commit i make! that way we can ensure that there will always be a value for all fields